docs/checkit.js

```diff
@@ -351,7 +351,7 @@ CheckIt.prototype.generateEmptyCalendar = function(calObj) {
         checkitApp.$calendarDiv.append('<div class="monthframe" id=' + monthObj.monthId + '></div>');
         
         if (monthObj.monthIndex === 0 || index === 0) {
-            var yearHeader = "<div class='page-header text-center' id='yearHeader'>" +
+            var yearHeader = "<div class='page-header text-center yearHeader'>" +
```

Comment:
`yearHeader` probably should have been spelled `year-header` based on the others here